index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section id="nav">
+    <div class="nav-bar">
+      <div class="menu-bar"></div>
```

Comment:
- [optional]  The logo text suppose to be wrap by a link `a` tag. This is for three reasons: convention, semantics, and user experience(UX). ℹ️